### PR TITLE
feat: readme rephrasing, clarifications and other minor improvements

### PR DESCRIPTION
**Pull request by kilrau**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,28 +2,26 @@
...
+**OpenDEX is a global network for unstoppable P2P Trading of cryptocurrencies and tokens.** At its core, it is a set of open protocol specifications, called **BOLD** (for Basics of L2 DEX). These specifications ensure that OpenDEX implementations remain compatible and users of different software implementations can trade with each other. **OpenDEX is not a company** or any other sort of legal entity, it is an open community of motivated individuals maintaining the BOLD specifications on GitHub. All node implementations following BOLD form the OpenDEX network. OpenDEX was started, because we believe there should be only one DEX, **one global trading network**; not hundreds. Find the slides from the first public announcement at #hcpp19 in Prague [here](201901005_hcpp19.pdf) and the video recording [here](https://www.youtube.com/watch?v=euSr9A6tI90).
```
Comment (hatmer) [inline README.md]: are we really a L2 Dex? I would think it's L3 since it's Bitcoin + Lightning/Raiden + Xud

Comment (kilrau) [reply README.md]: True that.

Comment (hatmer) [reply README.md]: also the lightning specs use "basis" rather than "basics", in case we want to conform

Comment (hatmer) [reply README.md]: "OpenDEX was started because we believe there should be only one DEX, **one global trading network**, not hundreds."

Comment (kilrau) [reply README.md]: Fixed.

**Review by hatmer — CHANGES_REQUESTED** [review]
looks great!

File: README.md (hunk 1 above)
Comment (kilrau) [reply README.md]: Forgot to push. Now.

File: README.md [hunk 2]
```diff
@@ -2,28 +2,26 @@
...
+**OpenDEX is a global network for unstoppable P2P Trading of cryptocurrencies and tokens.** At its core, it is a set of open protocol specifications, called **BOLD** (for Basis of L3 DEX). These specifications ensure that OpenDEX implementations remain compatible and users of different software implementations can trade with each other. **OpenDEX is not a company** or any other sort of legal entity, it is an open community of motivated individuals maintaining the BOLD specifications on GitHub. All node implementations following BOLD form the OpenDEX network. OpenDEX was started because we believe there should be only one DEX, **one global trading network**, not hundreds. Find the slides from the first public announcement at #hcpp19 in Prague [here](201901005_hcpp19.pdf) and the video recording [here](https://www.youtube.com/watch?v=euSr9A6tI90).
 
 ## Get Started
-
--> [Read the BOLD protocol specifications](BOLD-00.md)
+-> [Read the BOLD specifications](BOLD-00.md)
```
Comment (hatmer) [inline README.md]: let's change to 00-introduction.md and merge

Comment (kilrau) [reply README.md]: Good catch! Fixed. Merging.

**Pull request merged**